### DISCUSSION TITLE
How do I reset the colors for apps to the default?

### DISCUSSION THREAD

**Discussion by Quadragintillion** [end-4/dots-hyprland#688] [Q&A] [answered]
I recently installed this on top of Gnome, but the color scheme messes with windows on Gnome (makes the titlebar look strange and changes the color for things like the settings app) and I don't want that. Is there a way to reset the color scheme for windows entirely?

**Comment by end-4** [comment, marked as answer]
delete `~/.config/gtk-4.0/gtk.css`
alternatively, in hyprland with my dotfiles, use the `>adw` command in the launcher

**Reply by Quadragintillion** [reply]
> delete `~/.config/gtk-4.0/gtk.css` alternatively, in hyprland with my dotfiles, use the `>adw` command in the launcher

Everything is stuck in light mode
Edit: Nevermind, it reverted to dark after a minute
